rust_src/src/remacs_sys.rs

```diff
@@ -1369,6 +1377,8 @@ extern "C" {
     pub static Vprocess_alist: LispObject;
     pub static Vrun_hooks: LispObject;
 
+    pub fn casify_object(case_action: CaseAction, object: LispObject) -> LispObject;
```

Comment:
Typically we add to the bottom of the list of exported functions.

Reply:
Ok. it is moved now. If I got that wrong, feel free just to move to the right place. Thanks.